controllers/funding/views/publication-search.njk

```diff
@@ -1,12 +1,13 @@
 {% extends "layouts/main.njk" %}
-
 {% from "components/hero.njk" import hero with context %}
 {% from "components/content-box/macro.njk" import contentBox %}
 {% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
 {% from "components/promo-card/macro.njk" import promoCard %}
 {% from "components/split-nav/macro.njk" import splitNav %}
 {% from "components/icons.njk" import iconClose, iconTickPlain %}
 
+{# @TODO: Unscope this from insights section #}
```

Comment:
Didn't try to fix this in the PR, but we should update this so this copy isn't keyed by an unrelated section.